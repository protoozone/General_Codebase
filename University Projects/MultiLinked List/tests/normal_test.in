NEW 5
1
2.3
lol
p

NEW 4
{0}
{0}
abba
{0}
VIEW 0
VIEW 1
VIEW ALL
VIEW-NESTED 0
VIEW-NESTED 1
INSERT 0 0 b
VIEW-NESTED 0
VIEW-NESTED 1
DELETE 0 2
VIEW-NESTED 1
INSERT 1 3 {0}
VIEW-NESTED 1
TYPE 1
TYPE 0
DELETE 0 0
DELETE 0 3
VIEW-NESTED 1
NEW 6
lol
1
2
3.0
p
{0}
VIEW 2
VIEW-NESTED 2
TYPE 2
REMOVE 2
EXIT