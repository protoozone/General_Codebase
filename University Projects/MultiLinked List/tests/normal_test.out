List 0: 1 -> 2.30 -> lol -> p -> 
Nested 1: {List 0} -> {List 0} -> abba -> {List 0}
1 -> 2.30 -> lol -> p -> 
{List 0} -> {List 0} -> abba -> {List 0}
Number of lists: 2
List 0
Nested 1
1 -> 2.30 -> lol -> p -> 
{1 -> 2.30 -> lol -> p -> } -> {1 -> 2.30 -> lol -> p -> } -> abba -> {1 -> 2.30 -> lol -> p -> }
List 0: b -> 1 -> 2.30 -> lol -> p -> 
b -> 1 -> 2.30 -> lol -> p -> 
{b -> 1 -> 2.30 -> lol -> p -> } -> {b -> 1 -> 2.30 -> lol -> p -> } -> abba -> {b -> 1 -> 2.30 -> lol -> p -> }
List 0: b -> 1 -> lol -> p -> 
{b -> 1 -> lol -> p -> } -> {b -> 1 -> lol -> p -> } -> abba -> {b -> 1 -> lol -> p -> }
Nested 1: {List 0} -> {List 0} -> abba -> {List 0} -> {List 0}
{b -> 1 -> lol -> p -> } -> {b -> 1 -> lol -> p -> } -> abba -> {b -> 1 -> lol -> p -> } -> {b -> 1 -> lol -> p -> }
reference -> reference -> string -> reference -> reference
char -> int -> string -> char -> string
List 0: 1 -> lol -> p -> 
List 0: 1 -> lol -> p
{1 -> lol -> p} -> {1 -> lol -> p} -> abba -> {1 -> lol -> p} -> {1 -> lol -> p}
Nested 2: lol -> 1 -> 2 -> 3.00 -> p -> {List 0}
lol -> 1 -> 2 -> 3.00 -> p -> {List 0}
lol -> 1 -> 2 -> 3.00 -> p -> {1 -> lol -> p}
string -> int -> int -> float -> char -> reference
List 2 has been removed.

Number of lists: 2
List 0
Nested 1